### PR TITLE
Free tier changes 3/3

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This PR includes the final changes we need to make in preparation for the free tier changes coming on November 21. This is mostly just tweaking some download URLs so they don't read `cgr.dev/chainguard/...` and adding notes where relevant.

### What should this PR do?

resolves https://github.com/chainguard-dev/internal/issues/4161

### Why are we making this change?

The referenced images will no longer be free and we want to make sure that we're providing accurate information to our readers.

### How should this PR be tested?

Changes should make sense as written!

### CODE REVIEW

**Review by smythp — APPROVED** [review]
Looks generally good, had a couple comments and feel like we should leverage the site functionality more for these kinds of updates (i.e. stuff that mentions a date and will probably need to be updated/removed in 6-9 months when the free tier changes are a distant memory).

File: content/chainguard/chainguard-images/getting-started/cilium/index.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ We will demonstrate how to get started with the Chainguard Cilium images on an e
 {{< blurb/images >}}
 {{< /details >}}
 
+> **Note**: In November 2024, after this article was first written, Chainguard [made changes to its free tier of Developer Imgages](https://www.chainguard.dev/unchained/changes-to-chainguard-images-developer-tier). In order to access the Cilium images used in this guide, you will need to be part of an organization that has access to these images. For a full list of Developer Images that will remain in Chainguard's free tier, please refer to [this support page](https://support.chainguard.dev/hc/en-us/articles/28452542784667-Customer-Notice-Free-Image-Tier-Changes).
```
[the remark this answers is not in the document]
Comment (matthewhelmke) [reply content/chainguard/chainguard-images/getting-started/cilium/index.md:39]: I agree that we need to find a way as a team to decide on a standard method for all of us to create shortcode / blurbs like you describe. Okay, that's my unsolicited $0.02

[the remark this answers is not in the document]
Comment (smythp) [reply content/chainguard/chainguard-images/getting-started/cilium/index.md:39]: :) We do currently have a pretty OK way to do it (edu/layouts/shortcodes/blurb), then you can include them in text like this (assuming you Create a blurb called free-tier-message.html):

```
{{< blurb/free-tier-message >}}
```

Conventionally we've used them for definitions and wrapped them in a details drawer, like this, but for this one I think you'd use the bare blurb without the details.

```
{{< details "What is Deep Learning?" >}}
{{< blurb/deep-learning >}}
{{< /details >}}
```

I think we underutilize this functionality, looking in there I've created 7/13 of the blurbs and the only ones that really get used are the Wolfi, distroless, and Images blurbs, and some of the others are ancientenforce-era ones.

[the remark this answers is not in the document]
Comment (SharpRake) [reply content/chainguard/chainguard-images/getting-started/cilium/index.md:39]: Oof - don't know how I missed "Imgages"

Great point, I created a blurb for this.

**Comment by SharpRake** [comment]
I'm going to merge this now, I'd rather not wait on it.

**Pull request merged**